src/util/logging.cpp

```diff
@@ -241,4 +241,31 @@ void Logging::shutdown() {
     }
 }
 
+namespace {
+
+inline QString preambleString(const char* logContext) {
+    if ((logContext == nullptr) || (strlen(logContext) == 0)) {
```

Comment:
logContext is typical a string literal. Can we use QLatin1String interface instead?